hazelcast/src/main/java/com/hazelcast/query/impl/Indexes.java

```diff
@@ -34,6 +35,11 @@
     private final ConcurrentMap<String, Index> mapIndexes = new ConcurrentHashMap<String, Index>(3);
     private final AtomicReference<Index[]> indexes = new AtomicReference<Index[]>(EMPTY_INDEX);
     private volatile boolean hasIndex;
+    private SerializationService ss;
```

Comment:
Don't call it ss... Chris also has this habbit and I don't like it at all. It makes code difficult to read because I need to convert ss to serializationService.